.github/workflows/changelog.yml

```
@@ -0,0 +1,32 @@
+name: Generate CHANGELOG.md from workflows
+
+on:
+  push:
+    - main
```

actionlint (1.7.7)

"push" section is sequence node but mapping node is expected

(syntax-check)